### PR TITLE
ENH: add aspect radio to limits dialog

### PR DESCRIPTION
**Pull request by gamaanderson**
enhancement according to issue #138

### CODE REVIEW

**Comment by nguy** [comment]
In [this section](https://github.com/nguy/artview/blob/master/artview/components/plot_radar.py#L900) there is a test for airborne radar. Can you change:
```
if (radar.metadata['platform_type'] == 'aircraft_tail' or
    radar.metadata['platform_type'] == 'aircraft'):
```
to:
```
if 'aircraft' in radar.metadata['platform_type']:

File: artview/components/limits.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def _default_limits(field, plot_type):
 ###############################
```
Comment (nguy) [inline artview/components/limits.py]: In [L59-64](https://github.com/nguy/artview/blob/master/artview/components/limits.py#L59), we could set an aspect ratio for airborne or rhi identified files.

Comment (gamaanderson) [inline artview/components/limits.py]: current aspect ratio is defined as auto, I would be careful on changing that resizing may become problematic.

**Comment by gamaanderson** [comment]
see new commit to the airborne change.

**Comment by nguy** [comment]
The changes look good. Last thing is that I feel the other limits are pretty self-explanatory. But the aspect ratio may not be to people. Should we add a help button that pops up a window like we do in other areas?

**Comment by nguy** [comment]
@stechma2 See the image below. Would you know how to use this?
screen shot 2016-03-22 at 8 17 54 am

**Comment by gamaanderson** [comment]
what would be the help text. If it is not too long it could be a tool tip.

**Comment by nguy** [comment]
Oh, that's a great idea! Thoughts for tooltips:
Data Min: "Minimum data value to display"
Data Max: "Maximum data value to display"
X Min: "Minimum X-axis value"
X Max: "Maximum X-axis value"
Y Min: "Minimum Y-axis value"
Y Max: "Maximum Y-axis value"
Aspect Ratio: "Y-to-X size display ratio, also accepts auto and equal"

**Comment by stechma2** [comment]
@nguy I would know how to use that, though I think adding the tooltips would be very useful (I like the ones you suggested). Also, glad to see the file being recognized as airborne now with the auto-scaling applied. Thanks for these changes!

**Comment by gamaanderson** [comment]
I also like the suggestion. May be we could change X and Y for "horizontal" and "vertical", as to avoid confusion with the x and y direction in grid.

**Comment by gamaanderson** [comment]
just uploaded a commit with tooltips

**Comment by nguy** [comment]
Nice changes. I think this is ready for merge. Agreed?

**Comment by nguy** [comment]
@gamaanderson Can you make one more additional change I was just made aware of by a user?
In the README, under the install section. artview is mispelled as artivew. Since this will likely be merged soon.

**Comment by gamaanderson** [comment]
done, also rolled the micro version, as this has a functionality change (was that procedure correct?)
I'm ok with merging.

**Pull request merged**